static/src/javascripts/projects/common/modules/ui/subscription-banners/guardian-weekly-banner-template.js

```diff
@@ -0,0 +1,115 @@
+// @flow
+
+import closeCentralIcon from 'svgs/icon/close-large.svg';
+import theGuardianLogo from 'svgs/logo/the-guardian-logo.svg';
+
+const isUserLoggedIn = userLoggedIn =>
```

Comment:
I think this should be renamed - it sounds like it returns a boolean at the moment

Reply:
it does return a boolean

Reply:
`isUserLoggedIn` is a function with the signature `(boolean) => string`, so it returns a string unless I'm mistaken.

Reply:
`loggedInClassModifier` or similar would be more suitable IMO

Reply:
this is the `isUserLoggedIn` function I'm using `const isUserLoggedIn = (): boolean => getUserFromCookie() !== null`

Reply:
Yes that's the function you're calling in another file whose return value eventually gets put in the argument `userLoggedIn` here.

But the `const` function on line 6 of this file is called `isUserLoggedIn` and yet returns a string, either empty string `''` or `'site-message--subscription-banner__sign-in--already-signed-in'`.

I feel like we're getting our wires crossed, so leave this if you like and we can sort it later.